### ISSUE TITLE
CSV format

### ISSUE THREAD

**Issue by zmanion** [distributedweaknessfiling/DWF-Database#28]
What CSV editor/producer is anyone using? I've tried Excel, Excel with a macro, LibreOffice, comma-chameleon (yes, really), all on a Mac. I can't easily produce "all cells quoted unless integers." Hand editing is painful. Emacs csv-mode. Have not tried vi/vim + csv.
 
Even before better tool support (like code/forms that generate CSV rows and JSON files), can we agree on a DWF CSV format and/or editor?

**Comment by kurtseifried** [comment]
Agreed, unless someone strongly objects I will move the CSV to "ALL FIELDS MUST BE QUOTED" this weekend/early next week.
